### PR TITLE
Move dep install to install script.

### PR DESCRIPTION
**Pull request by russellb**
The repo sync script installed dep.  Move it to the dependency
installation script instead.  Also don't bother if dep is already
installed.

### CODE REVIEW

File: 01_install_requirements.sh [hunk 1]
```diff
@@ -183,3 +183,11 @@ if ! which operator-sdk 2>&1 >/dev/null ; then
     sudo wget https://github.com/operator-framework/operator-sdk/releases/download/v0.9.0/operator-sdk-v0.9.0-x86_64-linux-gnu -O /usr/local/bin/operator-sdk
     sudo chmod 755 /usr/local/bin/operator-sdk
 fi
+
+# Install Go dependency management tool
+# Using pre-compiled binaries instead of installing from source
+if ! which dep 2>&1 >/dev/null ; then
+    mkdir -p $GOPATH/bin
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+    export PATH="${GOPATH}/bin:$PATH"
```
Comment (hardys) [inline 01_install_requirements.sh]: For the `which` to work here, I think we have to move the `export PATH` out of the conditional, and also ensure GOPATH is set with eval go env?

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/912/

File: 01_install_requirements.sh (hunk 1 above)
Comment (russellb) [reply 01_install_requirements.sh]: yeah, sorry for not trying this

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/913/

**Comment by russellb** [comment]
> Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/913/

failed waiting on a worker to come online, so got past these changes

**Pull request merged**